testing/run_tests.py

```diff
@@ -351,6 +351,14 @@ def RunJavaTests(filter, android_variant='android_debug_unopt'):
   RunCmd(command, cwd=test_runner_dir, env=env)
 
 
+def RunAndroidTests(android_variant='android_debug_unopt'):
+  test_runner_name = 'flutter_shell_native_unittests'
+  tests_path = os.path.join(out_dir, android_variant, test_runner_name)
+  remote_path = '/data/local/tmp'
+  remote_tests_path = os.path.join(remote_path, test_runner_name)
+  RunCmd(['adb', 'push', tests_path, remote_path], cwd=buildroot_dir)
+  RunCmd(['adb', 'shell', remote_tests_path])
```

Comment:
How does this validate failure or pass?

Reply:
`adb shell` forwards the exit code which RunCmd grabs.